pkg/placement/membership.go

```diff
@@ -239,8 +267,14 @@ func (p *Service) disseminateOperation(targets []placementGRPCStream, operation
 	var err error
 	for _, s := range targets {
 		err = s.Send(o)
+
 		if err != nil {
-			log.Errorf("error updating host on unlock operation: %s", err)
+			peerAddr := "n/a"
+			if peer, ok := peer.FromContext(s.Context()); ok {
+				peerAddr = peer.Addr.String()
+			}
+
+			log.Errorf("error updating host(%s) on unlock operation: %s", peerAddr, err)
 			// TODO: the error should not be ignored. By handing error or retrying dissemination,
 			// this logic needs to be improved. Otherwise, the runtimes throwing the exeception
 			// will have the inconsistent hashing tables.
```

Comment:
The `continue` is redundant here. BTW, can you please add some comment why we return the last error and not failing fast?

Reply:
This should be failling fast and send some abort operation if we implement three phase commit concensus algorithm correctly. but that's the existing code behavior that I didn't touch yet. 

I do not want to change the existing behavior now. Instead, I left the `TODO` comment but will create the separate issue and revisit this dissemination algorithm.